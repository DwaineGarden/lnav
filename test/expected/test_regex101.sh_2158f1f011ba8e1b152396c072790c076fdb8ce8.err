[33m‚ö† warning[0m: not deleting regex101 entry ‚Äú[1mzpEnjV[0m‚Äù
   [33mreason[0m: delete code is not known for this entry
[36m =[0m   [36mnote[0m: formats created by importing a regex101.com entry will not have a delete code
