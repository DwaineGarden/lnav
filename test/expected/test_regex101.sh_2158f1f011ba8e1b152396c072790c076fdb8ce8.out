[1m[32mâœ” [0mdeleted regex101 entry: [1mzpEnjV[0m
